content/en/boilerplates/mc-env-var.md

```diff
@@ -0,0 +1,54 @@
+---
+---
+To simplify the deployment instructions and use the examples provided, set the
+following environment variables. If you use your own string values, ensure they
+are viable as filenames and compatible with `Makefiles`. Go to the [GNU guide](https://www.gnu.org/software/make/manual/html_node/Rule-Syntax.html)
+for more information.
+
+The guide assumes that the following environment variables are set:
+
+<table>
+   <thead>
+      <tr>
+      <th><strong>Environment Variable</strong></th>
+      <th><strong>Description</strong></th>
+      </tr>
+   </thead>
+      <tbody>
+      <tr>
+         <td>ISTIO</td>
+         <td>The top-level directory of the Istio installation.</td>
+      </tr>
+      <tr>
+         <td>CLUSTER_1</td>
+         <td>The name of the first cluster. The guide uses <code>cluster1</code>.</td>
+         </tr>
+      <tr>
+         <td>CLUSTER_2</td>
+         <td>The name of the second cluster. Must be different from the value in
+         <code>CLUSTER_1</code>. The guide uses <code>cluster2</code>.</td>
+         </tr>
+      <tr>
+         <td>CTX_1</td>
+         <td>The path to the Kubernetes context file for the first cluster.</td>
+      </tr>
+      <tr>
+         <td>CTX_2</td>
+         <td>The path to the Kubernetes context file for the second cluster.</td>
+      </tr>
+      <tr>
+         <td>NETWORK_1</td>
+         <td>The name of the first network. The guide uses <code>network1</code>.</td>
```

Comment:
This and mesh id I expect to be very confusing. I would not know what to set here if I was not familiar already

Reply:
Familiarity with single cluster deployments is required for this content.